firebase-firestore/src/androidTest/java/com/google/firebase/firestore/POJOTest.java

```diff
@@ -301,19 +301,19 @@ public void testCantWriteNonStandardNumberTypes() {
     pojo.bigIntegerValue = new BigInteger("0");
     expectedErrorMessages.put(
         pojo,
-        "Could not serialize object. BigInteger is not supported, please use an int, long, float or double (found in field 'bigIntegerValue')");
+        "Could not serialize object. Numbers of type BigInteger are not supported, please use an int, long, float or double (found in field 'bigIntegerValue')");
```

Comment:
So remember in that other PR? Where I was talking about over-specified error message checking in tests? :)

(This is still fine; no action required.)

Reply:
FWIW- As a reviewer, I actually like that I can just look at the tests to see the exact error message that comes out without worrying that we're missing a space somewhere or that some generated part of the message is coming out goofy.

Also, as a reviewer, I should actually check if the tests pass before merging PRs.
![box_of_shame](https://user-images.githubusercontent.com/206364/54369587-6ac3bd00-4633-11e9-8e1d-4b22a95891e9.gif)

(in any case, there's definitely a balance to be struck with these sorts of tests... if we feel they're too fragile, I don't mind revisiting)

Reply:
I actually am a big fan of having at least one place that checks the full error message. A lot of our error messages are composed of different parts, and it is very easy to omit words/spaces or put extra spaces when we do this.